docs/products/cassandra/howto/disable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,117 @@
+===============================================================================
+
+.. important::
+
+   This private beta feature is available on request only. Contact sales@Aiven.io or support@Aiven.io to opt in. 
+
+This article provides you with instructions on how to discontinue the cross-cluster replication (CCR) for your Aiven for Apache Cassandra® service.
+
+About disabling CCR
+-------------------
+
+When you enable CCR for your service, you connect it to another service (replica service), which results in creating a CCR pair of services. You can disable CCR for your service either by splitting the services constituting the CCR pair or by deleting one of them.
+
+* `Aiven console <https://console.aiven.io/>`_
+* CLI
+* API
+
+It's recommended to use the Aiven console for disabling CCR.
+
+Prerequisites
+-------------
+
+* Aiven account
+* Depending on the method you choose to use for enabling CCR
+
+  * Access to the `Aiven console <https://console.aiven.io/>`_
+  * `cURL` CLI tool
+  * `Aiven CLI tool <https://github.com/aiven/aiven-client>`_
+
+* CCR enabled on a pair of Aiven for Apache Cassandra services
+
+Disable CCR in the console
+--------------------------
+
+In the console, use either of the two following methods to disable CCR on your service(s): split the services or delete one of them.
+
+Split the services
+''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select the service for which you'd like to disable CCR.
+3. In the **Overview** tab of the service's page, navigate to **Cross Cluster Replication** and select **Split cluster**.
+
+   .. warning::
+
+      As soon as you split the cluster, the two services constituting the CCR pair become independent. It's not possible to recreate the CCR pair with the same service as a replica. You can select any other Aiven for Cassandra service for that purpose.
+   
+4. When in the **Warning** popup, make sure you understand the impact and select **Split cluster**.
+
+.. topic:: Result
+
+   You service no longer has a cross-cluster replica.
+
+Delete a service
+''''''''''''''''
+
+To disable CCR on your service, delete the replica service that is connected to your services for CCR purposes.
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of the service's page, navigate to the **Cross Cluster Replication** section and select the name of the replica service provided in the CCR description, which is supposed to take you to the replica service's page.
+4. In the **Overview** tab of the replica service, select **Delete service** from the top right corner.
+
+   .. warning::
+
+      As soon as you delete the replica service, CCR gets disabled for your service that is now connected to the replica service for CCR purposes.
```

Comment:
> for your service that is now connected to the replica service for CCR purposes.

I think it's no longer connected to anything, after you deleted the other one?... I'd just end it at "CCR gets disabled" or maybe "your data is no longer replicated between regions"